File: src/main/scala/li/cil/oc/integration/enderstorage/ModEnderStorage.scala
package li.cil.oc.integration.enderstorage

import li.cil.oc.api.Driver
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModEnderStorage extends ModProxy {
  override def getMod = Mods.EnderStorage

  override def initialize() {
    Driver.add(new DriverFrequencyOwner)
  }
}

File: src/main/scala/li/cil/oc/integration/enderstorage/ModEnderStorage.java
package li.cil.oc.integration.enderstorage;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.util.mods.Mods;

public final class ModEnderStorage implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.EnderStorage();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverFrequencyOwner());
    }
}
